Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -134,6 +134,20 @@ open class NavigationMapView: UIView {
         }
     }
     
+    /**
+     Toggles displaying alternative routes when in active navigation session.
+     
+     If enabled, view will draw actual alternative route lines on the map.
+     Default value is `true`.
+     */
+    public var showsContinuousAlternatives: Bool = true {
```

Comment:
As far as I can tell, there’s nothing continuous about each alternative route individually; rather, the alternative routes are calculated continuously, just as proactive rerouting calculates routes proactively. This property should be named `showsAlternativeRoutesContinuously`.

Reply:
'Continuous alternatives' is the feature name we are trying to use publicly. It should also be different to just "alternative route" which is any "non-main" route set in Route Response. I tried to keep these terms apart to avoid confusion